cirq-google/cirq_google/serialization/circuit_serializer_test.py

```diff
@@ -1026,6 +1028,106 @@ def test_custom_serializer(use_constants_table: bool):
     assert op.qubits == (cirq.q(0, 0),)
 
 
+class DiscountTag:
+    def __init__(self, discount: float):
+        self.discount = discount
+
+    def __eq__(self, other):
+        if isinstance(other, DiscountTag) and self.discount == other.discount:
+            return True
+
+    def __hash__(self):
+        return hash(self.discount)
+
+
+class DiscountTagSerializer(TagSerializer):
+    """Describes how to serialize DiscountTag."""
+
+    def can_serialize_tag(self, tag):
+        return isinstance(tag, DiscountTag)
+
+    def to_proto(
+        self,
+        tag: Any,
+        msg: Optional[v2.program_pb2.CircuitOperation] = None,
+        *,
+        constants: List[v2.program_pb2.Constant],
+        raw_constants: Dict[Any, int],
+    ) -> v2.program_pb2.CircuitOperation:
```

Comment:
Typo in annotation -


```suggestion
        msg: Optional[v2.program_pb2.Tag] = None,
        *,
        constants: List[v2.program_pb2.Constant],
        raw_constants: Dict[Any, int],
    ) -> v2.program_pb2.Tag:
```

Reply:
Done.